guides/source/command_line.md

```diff
@@ -65,6 +65,8 @@ If you wish to skip some files or components from being generated, you can appen
 | `--skip-system-test`    | Skip system test files                                      |
 | `--skip-bootsnap`       | Skip bootsnap gem                                           |
 
+INFO: These are just some of the options you can append to `rails new`. For a comprehensive list of options, type `rails new --help`.
```

Comment:
Let's skip the `INFO:` callout here.  I don't think it adds anything, and too many callouts in a guide can be distracting.

```suggestion
These are just some of the options that `rails new` accepts. For a full list of options, type `rails new --help`.
```